### PR TITLE
e2e/test rtl support

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: e2e/cypress/support/commands.js [hunk 1]
```diff
@@ -6,8 +6,17 @@ import { INLINE_TOOLBAR_BUTTONS } from '../dataHooks';
 const resizeForDesktop = () => cy.viewport('macbook-15');
 const resizeForMobile = () => cy.viewport('iphone-5');
 
+const buildQuery = params => {
```
Comment (shaulgo) [inline e2e/cypress/support/commands.js]: very nice

File: e2e/test-env/src/server/renderer.jsx [hunk 2]
```diff
@@ -18,8 +18,9 @@ export default function renderer() {
   return (req, res) => {
     const [componentId, fixtureName = 'empty'] = req.path.replace(/^\/|\/$/g, '').split('/');
     const isMobile = req.query.mobile === '';
+    const locale = req.query.he === '' ? 'he' : 'en';
```
Comment (shaulgo) [inline e2e/test-env/src/server/renderer.jsx]: what do you think of `req.query.hebrew`?

File: e2e/test-env/src/shared/components/Editor/Editor.jsx [hunk 3]
```diff
@@ -78,7 +102,23 @@ class Editor extends Component {
...
+          locale={this.props.locale}
         />
+        <ReactModal
+          isOpen={this.state.showModal}
+          contentLabel="External Modal Example"
+          style={modalStyles}
+          role="dialog"
+          onRequestClose={onRequestClose || this.helpers.closeModal}
+        >
+          {this.state.showModal && (
```
Comment (shaulgo) [inline e2e/test-env/src/shared/components/Editor/Editor.jsx]: Is this needed?

Comment (mharsat) [reply e2e/test-env/src/shared/components/Editor/Editor.jsx]: copied as is from the example, removed it now from both

File: e2e/cypress/support/commands.js [hunk 4]
```diff
@@ -61,7 +79,10 @@ Cypress.Commands.add('newLine', () => {
 });
 
 Cypress.Commands.add('blurEditor', () => {
-  getEditor().blur();
+  getEditor()
+    .blur()
+    .get('[data-hook=inlineToolbar]')
```
[the remark this answers is not in the document]
Comment (mharsat) [reply e2e/cypress/support/commands.js]: this is the recommended cypress pattern for making sure an operation completed before proceeding. particularly when the editor is blurred, the inline toolbar may not disappear instantly leading to a flakey behavior, and so it waits for the operation to finish

**Pull request merged**